tgarchive/sync.py

```diff
@@ -142,6 +135,20 @@ def _get_messages(self, group, offset_id, ids=None) -> Message:
                 media=med
             )
 
+    def _fetch_messages(self, group, offset_id, ids=None) -> Message:
+        while True:
```

Comment:
Is this wait necessary? Isn't this handled [here](https://github.com/knadh/tg-archive/blob/d4d6e2db70ca9d7166015d02d2ce6eb8fa08529c/tgarchive/sync.py#L54)

Reply:
I thought handling the `FloodWaitError` here would be better rather than exiting when it happens. But it is unnecessary given  that we do have fetch_wait which can be set appropriately. So, just catching any exception and logging it should be good right? Or `_get_messages` doesn't do a try...except for a reason that I am missing out?

Reply:
Handling `FloodWaitError` is fine. I was referring to the `while` loop since there's already an external `white` loop. Needn't have both and can be refactored to have it in just one appropriate place.